app/views/clients/rollup/_custom_services.html.haml

```diff
@@ -0,0 +1,8 @@
+= cache([@client, 'enrollments', can_view_enrollment_details?, controller_path, current_user], expires_in: 1.hours) do
```

Comment:
I'm not sure about what needs to be in this cache key @eanders  can you take a look?

I think it should say something unique/descriptive like `'custom_services'` instead of `'enrollments'`.

Reply:
pretty sure that should be 
```suggestion
= cache([@client, 'custom_services', can_view_enrollment_details?, controller_path, current_user], expires_in: 1.hours) do
```

Reply:
Is there a good way to test the caching behavior other than just playing around with it? (e.g. how graphql apollo has dev tools where you can check out what's in the cache?)

For example, I noticed while testing the confidential project names (below thread) that when I updated a project to be confidential, I could still see the name until I cleared out this cache, after which it showed up as 'Confidential Project' which I think is the intended behavior. I was wondering if that means we need to add Projects to this cache key, or if that is such an infrequent use case that it's not worthwhile.